python3.8/rockcraft.yaml

```diff
@@ -0,0 +1,39 @@
+name: python
+version: '3.8'
+summary: Python 3.8
+description: |
+    Python is an interpreted, interactive, object-oriented, open-source
+    programming language.
+license: GPL-3.0
+
+base: bare
+build-base: ubuntu@20.04
+
+run-user: _daemon_
+
+platforms:
+    amd64:
+
+services:
+    python3:
+        startup: disabled
+        override: replace
+        command: python3
```

Comment:
This isn't a service, as is. Needs an optional application

Reply:
I thought about 

```
python3 [ app.py ]
```

But that would mean we need to inject a dummy `app.py` file in the rock as a placeholder. I am not sure if keeping the command like this without the file would be a good idea since the service will fail.

And right now, you can pass arguments to the any python script using `--args python3 /path/to/file.py` when you run it. Let me know what you think.

Reply:
The command looks good! I will use that.

> you can pass args even if [] isn't in the command?

Yup, you can.

Reply:
TIL